OracleIdentityGovernance/samples/externalDB/docker-compose.yaml

```diff
@@ -32,6 +32,11 @@ services:
       - DB_PASSWORD=${DC_ORCL_SYSPWD}
       - DB_SCHEMA_PASSWORD=${DC_RCU_SCHPWD}
       - RCUPREFIX=${DC_RCU_OIMPFX}
+    healthcheck:
+      test: ["CMD", "curl", "-fL", "http://oimadmin:7001/console"]
```

Comment:
If you're allowing the hostname to change, wouldn't this test need to change as well?